src/Blockcore.Indexer.Core/Storage/Mongo/MongoData.cs

```diff
@@ -511,15 +511,15 @@ public QueryResult<QueryAddressItem> AddressHistory(string address, int offset,
          // This will first perform one db query.
          long total = addressComputedTable.CountSent + addressComputedTable.CountReceived + addressComputedTable.CountStaked + addressComputedTable.CountMined;
 
-         if (offset == 0 || offset > total)
```

Comment:
if you remove this how will you know to get the top? we need something to indicate to fetch from the top